components/card/card.js

```diff
@@ -1,7 +1,6 @@
 export function createCharacterCard(url, status, type, occurrences) {
```

Comment:
As you are only exporting this function, I recommend to use an `export default` here.